src/Tizen.NUI.Scene3D/src/public/Controls/Panel.cs

```diff
@@ -0,0 +1,377 @@
+        /// If it is true, shadows are drawn on this panel.
+        /// </summary>
+        /// <param name="receiveShadow">Whether this Panel receives shadow or not.</param>
+        private void ReceiveShadow(bool receiveShadow)
+        {
+            Interop.Panel.ReceiveShadow(SwigCPtr, receiveShadow);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Retrieves whether the Panel receives shadow or not for Light
+        /// If it is true, this model is drawn on Shadow Map.
+        /// </summary>
+        /// <returns>True if this model receives shadow.</returns>
+        private bool IsShadowReceiving()
+        {
+            var isShadowReceiving = Interop.Panel.IsShadowReceiving(SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return isShadowReceiving;
+        }
+
+        /// <summary>
+        /// Sets defined resolution to the Panel.
+        /// The resolution is independent from the Panel size property.
+        /// The resolution defines a plane that the 2D UI scene will be rendered.
+        /// And the shape of the panel plane is defined by aspect ratio of the input resolution.
+        /// </summary>
+        /// <param name="resolution">The resolution defines panel plane.</param>
+        /// <returns>True if this model receives shadow.</returns>
+        private void SetPanelResolution(Vector2 resolution)
+        {
+            Interop.Panel.SetPanelResolution(SwigCPtr, Vector2.getCPtr(resolution));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Retrieves panel resolution.
+        /// </summary>
+        /// <returns>Resolution of the panel plane.</returns>
+        private Vector2 GetPanelResolution()
+        {
+            Vector2 resolution = new Vector2(Interop.Panel.GetPanelResolution(SwigCPtr), true);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return resolution;
+        }
+
+        /// <summary>
+        /// Sets root Actor of 2D UI content.
+        /// The content is rendered on the plane of the Panel by using off screen rendering.
+        /// </summary>
+        /// <param name="content">Root View of 2D UI content.</param>
+        private void SetContent(View content)
+        {
+            Interop.Panel.SetContent(SwigCPtr, content.SwigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Retrieves root View of 2D UI content.
+        /// </summary>
+        /// <returns>Root View of 2D UI content.</returns>
+        private View GetContent()
```

Comment:
```suggestion
        internal View GetContent()
```

Maybe internal? (It might be required some other classes inside of this repo)

Reply:
public Content property getter does samething.
I don't hope to use it in other classes...